http_cache.rst

```diff
@@ -157,6 +157,32 @@ When you're in debug mode (the second argument of ``Kernel`` constructor in the
 front controller is ``true``), Symfony automatically adds an ``X-Symfony-Cache``
 header to the response. Use this to get information about cache hits and misses.
 
+.. tip::
+
+    The ``trace_level`` config option can be set to ``none``, ``short``
+    and ``full`` to get different levels of information. When
+    configured explicitly, it works independent of the debug mode.
+
+    ``short`` contains the information for the master request only.
+    It's written in a concise way that makes it easy to record the
+    information in your server log files. For example, in Apache you can
+    use ``%{X-Symfony-Cache}o`` in ``LogFormat`` format statements.
```

Comment:
I would add "This information can be used to extract general information about cache efficiency of your routes." here instead of the new paragraph, to avoid a wall of text.